internal/tester/data.go

```diff
@@ -54,6 +54,12 @@ const (
 	//
 	// TODO: make configurable
 	PeriodicLoggingFrequency = 10 * time.Second
+
+	// EndAtTipCheckInterval is the frequency that EndAtTip condition
+	// is evaludated
+	//
+	// TODO: make configurable
+	EndAtTipCheckInterval = 10 * time.Second
```

Comment:
Since this doesn't change (and I don't imagine it will), can we move this to the stateful syncer and not pass as an arg?

Reply:
fair point 👍